.github/workflows/build-and-publish-rule-book.yaml

```diff
@@ -25,51 +25,31 @@ jobs:
       matrix:
         include:
           - language: en
-            version: Version
-            warning: "This is an unreleased version built on GitHub from commit"
             sections: "sections"
             luaengine: false
           - language: pl
-            version: Wersja
-            warning: "To jest nieopublikowana wersja zbudowana na GitHubie z commita"
             sections: "sections/translated/pl"
             luaengine: false
           - language: es
-            version: Versión
-            warning: "Se trata de una versión inédita creada en GitHub a partir del commit"
             sections: "sections/translated/es"
             luaengine: false
           - language: fr
-            version: Version
-            warning: "Ceci est une version non-éditée créée sur GitHub à partir du commit"
             sections: "sections/translated/fr"
             luaengine: false
           - language: ru
-            version: Версия
-            warning: "Данная версия является неопубликованной сборкой из коммита в GitHub"
             sections: "sections/translated/ru"
             luaengine: true
           - language: ua
-            version: Версія
-            warning: "Це неопублікована версія, зібрана на GitHub з коміту"
             sections: "sections/translated/ua"
             luaengine: true
           - language: de
-            version: Version
-            warning: "Dies ist eine unveröffentlichte Version auf GitHub"
             sections: "sections/translated/de"
             luaengine: true
 
     steps:
       - name: Set up repository
         uses: actions/checkout@v4
 
-      - name: Replace rule book version with a warning
```

Comment:
Why? We want this.

Reply:
Ah, ok, you put it in the latex code.

Reply:
Thanks, I was going to move that warning over as well, to have all the strings (which are not in .po) translated in the same place